libraries/SITL/examples/Airsim/quadX.parm

```diff
@@ -1,4 +1,4 @@
-ARMING_CHECK     0.000000
+ARMING_CHECK     1.000000
```

Comment:
This is the default value so we should just remove it from param file.

Reply:
Done
Thanks for the review!